src/State/State.js

```diff
@@ -0,0 +1,28 @@
+
+import React, { useState } from 'react'
+import Context from '../context/NoteContext';
```

Comment:
You don't need to create a separate file just for creating a context. Create NoteContext here in this file and remove the `NoteContext` file.